### PR TITLE
Add error logging

### PR DESCRIPTION
**Pull request by elishaking**
- Implemented Error Logging to Sentry (limited to action-creators for now)

### CODE REVIEW

File: src/utils/logError.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import { captureException } from "@sentry/browser";
+
+export default function logError(err: any) {
+  captureException(err);
```
Comment (skaex) [inline src/utils/logError.ts]: In development, you don't want to log to sentry, you maybe want to log to the console. You can check the environment here.

Comment (elishaking) [reply src/utils/logError.ts]: ok, I'll look into that

Comment (elishaking) [reply src/utils/logError.ts]: I just finished working on this

File: src/utils/logError.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { captureException } from "@sentry/browser";
+
+export default function logError(err: any) {
+  if (process.env.NODE_ENV === "development") {
+    console.error(err);
+  } else if (process.env.NODE_ENV === "production") {
+    captureException(err);
+  }
```
Comment (skaex) [inline src/utils/logError.ts]: What happens on `else`? That is when none of the two cases is true.

**Pull request merged**